tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/sequencefile/SequenceFileScanner.java

```diff
@@ -105,6 +105,11 @@ public void init() throws IOException {
     }
 
     String delim  = meta.getProperty(StorageConstants.SEQUENCEFILE_DELIMITER, StorageConstants.DEFAULT_FIELD_DELIMITER);
+    if (delim == null || delim.isEmpty()) {
+      delim = meta.getProperty(StorageConstants.TEXT_DELIMITER, StorageConstants.DEFAULT_FIELD_DELIMITER);
+    }
+    meta.getPropertySet().set(StorageConstants.TEXT_DELIMITER, delim);
```

Comment:
The property order should be TEXT_DELIMITER, SEQUENCEFILE_DELIMITER and please use default value of getProperty.
Why you set the property ? is there any reason ?